packages/react-native/ReactAndroid/src/main/java/com/facebook/react/defaults/DefaultReactHost.kt

```diff
@@ -60,7 +59,7 @@ public object DefaultReactHost {
       jsMainModulePath: String = "index",
       jsBundleAssetPath: String = "index",
       jsBundleFilePath: String? = null,
-      isHermesEnabled: Boolean = true,
+      jsRuntimeFactory: JSRuntimeFactory,
```

Comment:
That's a breaking change that will impact all the apps in React Native and can't be shipped as it is.